test/validation/org/apache/cassandra/bridges/ctoolbridge/CToolBridge.java

```diff
@@ -85,6 +114,23 @@ public void updateConf(Map<String, String> options)
             execute("ctool change_config %s all --k %s --value %s", DEFAULT_CLUSTER_NAME, key, options.get(key));
     }
 
+    public boolean checkNodeCount(int nodes)
+    {
+        String[] endpoints = clusterEndpoints();
+        if(endpoints.length != nodes)
+        {
+            return false;
+        }
+        return true;
```

Comment:
This entire function can be condensed to `return nodes == clusterEndPoints().length`